pkg/services/ngalert/api/tooling/Makefile

```diff
@@ -26,7 +26,7 @@ ensure_go-swagger_mac:
 	@hash swagger &>/dev/null || (brew tap go-swagger/go-swagger && brew install go-swagger)
 
 spec.json-mac: ensure_go-swagger_mac $(GO_PKG_FILES)
-	swagger generate spec -m -w $(API_DIR) -o spec.json && swagger generate spec -m --include-tag=stable -o ngalert.json
+	SWAGGER_GENERATE_EXTENSION=false swagger generate spec -m -w $(API_DIR) -o spec.json && swagger generate spec -m --include-tag=stable -o ngalert.json
```

Comment:
we need to set the same for [the target using docker](https://github.com/grafana/grafana/blob/9a0ba6bfef514f7348f9451923f6786c46e0b03d/pkg/services/ngalert/api/tooling/Makefile#L14) for generating the specification (for consistent generated output on both cases)

this is required otherwise definitions that are defined both in the [alerting](https://github.com/grafana/grafana/blob/e9f8d582c8084ade23fa6c4791e986dc9992f713/pkg/services/ngalert/api/tooling/api.json#L370) and in the [api-spec.json](https://github.com/grafana/grafana/blob/e9f8d582c8084ade23fa6c4791e986dc9992f713/public/api-spec.json#L8753) specification are reported as different just because of the missing `x-go-name` properties.